u8timeseries/models/arima.py

```diff
@@ -72,6 +72,7 @@ def __init__(self, start_p=1, max_p=12, start_q=0, max_q=12, max_P=2, max_Q=2, s
     def __str__(self):
         return 'auto-ARIMA'
 
+    @time_log(logger=logger)
```

Comment:
I vote for removing these time logging in models. Printing progress can be useful (when possible), but logging time should be up to the user I think.

Reply:
Should I do it as part of this branch?

Reply:
No not needed